src/language-graphql/printer-graphql.js

```diff
@@ -128,7 +128,12 @@ function genericPrint(path, options, print) {
         return [
           '"""',
           hardline,
-          join(hardline, node.value.replace(/"""/g, "\\$&").split("\n")),
+          join(hardline,
+            node.value
+            .replace(/"""/g, "\\$&")
+            .split("\n")
+            .map((val, idx, arr) => arr.length === 1 ? val.trim() : val)
```

Comment:
Only trim value when it's a single line description, so it doesn't affect multi-line descriptions